### PR TITLE
Exchange inline-block to flexbox, add fonts and footer-block

### PR DESCRIPTION
**Pull request by MariaOdinson**

### CODE REVIEW

File: styles/style.css [hunk 1]
```diff
@@ -22,21 +34,40 @@
   border: 0;
 }
 
-.inline-block {
-  display: inline-block;
+.item {
   width: 300px;
   height: 180px;
   margin: 80px 8px;
```
Comment (atenobi) [inline styles/style.css:41]: here you margin is can be different for better result in pixel perfect

File: styles/style.css [hunk 2]
```diff
@@ -22,21 +34,40 @@
...
   box-sizing: border-box;
 }
 
-.image-block {
-  height: 100px;
-  width: 100px;
+.image {
```
Comment (atenobi) [inline styles/style.css:46]: Iam not agree with your desigion to remove width and height parameters for image

File: index.html [hunk 3]
```diff
@@ -6,38 +6,67 @@
...
       </div>
+    </div>
 
-      <div class="inline-block">
-        <div class="image-block">
-          <img src="./images/reviews.png" alt="cool" />
+    <div class="footer">
+      <div class="footer-container">
+        <div class="about">
+          <p>About</p>
```
Comment (atenobi) [inline index.html]: if you look carefuly in figma -you fid out that element have some spesial font-weight parameter, so you need class for that

File: styles/style.css [hunk 4]
```diff
@@ -22,21 +34,40 @@
...
 .description {
   color: #383838;
   font-size: 24px;
+  margin-top: 48px;
+}
+.footer {
+  background-color: #10367c;
+  color: #ffffff;
+  overflow: auto;
+  font-size: 20px;
```
Comment (atenobi) [inline styles/style.css]: i think - not better solution

File: index.html [hunk 5]
```diff
@@ -6,38 +6,67 @@
...
+          <p>How Triphouse works</p>
```
Comment (atenobi) [inline index.html]: you need also some class for p elements

File: styles/style.css [hunk 6]
```diff
@@ -22,21 +34,40 @@
...
+}
+
+.footer-container {
+  display: flex;
+  justify-content: space-around;
+  max-width: 1440px;
+  box-sizing: border-box;
+  margin: 120px auto 64px;
```
Comment (atenobi) [inline styles/style.css]: think about padding for .footer)))

File: styles/style.css [hunk 7]
```diff
@@ -22,21 +34,40 @@
...
+.footer {
+  background-color: #10367c;
+  color: #ffffff;
+  overflow: auto;
+  font-size: 20px;
+}
+
+.footer-container {
+  display: flex;
+  justify-content: space-around;
```
Comment (atenobi) [inline styles/style.css]: i am not sure:
space-around
margin?

File: index.html (hunk 5 above)
Comment (MariaOdinson) [reply index.html]: fixed

File: styles/style.css (hunk 2 above)
Comment (MariaOdinson) [reply styles/style.css:46]: fixed

File: index.html (hunk 3 above)
Comment (MariaOdinson) [reply index.html]: fixed

**Pull request merged**